test/unit/org/apache/cassandra/sensors/SensorsWriteTest.java

```diff
@@ -274,6 +287,92 @@ public void testSingleCounterMutation() throws WriteTimeoutException
         assertResponseSensors(localSensor.getValue(), registrySensor.getValue(), CF_COUTNER);
     }
 
+    @Test
+    public void testLWTPrepare() {
+        store = SensorsTestUtil.discardSSTables(KEYSPACE1, CF_STANDARD);
+        Context context = new Context(KEYSPACE1, CF_STANDARD, store.metadata.id.toString());
+        PartitionUpdate update = new RowUpdateBuilder(store.metadata(), 0, "0")
+                                 .add("val", "0")
+                                 .buildUpdate();
+        Commit proposal = Commit.newPrepare(update.partitionKey(), store.metadata(), UUIDGen.getTimeUUID());
+        handlePaxosPrepare(proposal);
+
+        Sensor writeSensor = SensorsTestUtil.getThreadLocalRequestSensor(context, Type.WRITE_BYTES);
+        assertThat(writeSensor.getValue()).isGreaterThan(0);
+        Sensor registryWriteSensor = SensorsTestUtil.getRegistrySensor(context, Type.WRITE_BYTES);
+        assertThat(registryWriteSensor).isEqualTo(writeSensor);
+        assertResponseSensors(writeSensor.getValue(), registryWriteSensor.getValue(), CF_STANDARD);
+
+        // handle the commit again, this time paxos has state because if the first proposal and read bytes will be populated
+        handlePaxosPrepare(proposal);
+        Sensor readSensor = SensorsTestUtil.getThreadLocalRequestSensor(context, Type.READ_BYTES);
```

Comment:
Shall we then check that we had no `READ_BYTES` sensor (or with value zero) for the previous request?